provider_packages/prepare_provider_packages.py

```diff
@@ -253,91 +228,59 @@ def get_package_release_version(provider_package_id: str,
         backport_packages=backport_packages).release_version + version_suffix
 
 
-def do_setup_package_providers(provider_package_id: str,
-                               version_suffix: str,
-                               package_dependencies: Iterable[str],
-                               extras: Dict[str, List[str]],
-                               backport_packages: bool) -> None:
+def get_install_requirements(
+        provider_package_id: str,
+        backport_packages: bool) -> List[str]:
     """
-    The main setup method for package.
+    Returns install requirements for the package.
 
     :param provider_package_id: id of the provider package
-    :param version_suffix: version suffix to be added to the release version (for example rc1)
-    :param package_dependencies: dependencies of the package
-    :param extras: extras of the package
-    :param backport_packages: whether to prepare regular (False) or backport (True) packages
+    :param backport_packages: whether we are preparing backport packages
 
+    :return: install requirements of the package
     """
-    setup.write_version()
-    provider_package_name = get_pip_package_name(provider_package_id, backport_packages=backport_packages)
-    package_name = f'{provider_package_name}'
-    package_prefix = f'airflow.providers.{provider_package_id}'
-    found_packages = find_packages()
-    found_packages = [package for package in found_packages if package.startswith(package_prefix)]
+    dependencies = PROVIDERS_REQUIREMENTS[provider_package_id]
 
     if backport_packages:
         airflow_dependency = 'apache-airflow~=1.10' if provider_package_id != 'cncf.kubernetes' \
             else 'apache-airflow>=1.10.12, <2.0.0'
     else:
         airflow_dependency = 'apache-airflow>=2.0.0a0'
+    install_requires = [airflow_dependency]
+    install_requires.extend(dependencies)
+    return install_requires
 
-    install_requires = [
-        airflow_dependency
+
+def get_setup_requirements() -> List[str]:
+    """
+    Returns setup requirements (common for all package for now).
+    :return: setup requirements
+    """
+    return [
+        'bowler',
+        'docutils',
+        'gitpython',
+        'setuptools',
+        'semver',
```

Comment:
```suggestion
```

Nor semver?

Reply:
Same here. We only need semver now in the prepare* script. The setup.py we generate now from the jinja2 template is pretty simple and generic.

Reply:
It's all done. And even removed docutils and gitpython for now, since we are not using them. We might add them later if we split the docs per-provider #11423